README.md

```diff
@@ -1,201 +1,256 @@
+[![codecov](https://codecov.io/gh/DataBiosphere/data-store/branch/master/graph/badge.svg)](https://codecov.io/gh/DataBiosphere/data-store)
 
-[![](https://img.shields.io/badge/slack-%23data--store-557EBF.svg)](https://humancellatlas.slack.com/messages/data-store/)
-[![Build Status](https://travis-ci.com/HumanCellAtlas/data-store.svg?branch=master)](https://travis-ci.com/HumanCellAtlas/data-store)
-[![codecov](https://codecov.io/gh/HumanCellAtlas/data-store/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/data-store)
-
-This repository maintains the data storage system of the
-[Human Cell Atlas](https://www.humancellatlas.org/). We use this
+This repository maintains the data storage system. We use this
 [Google Drive folder](https://drive.google.com/open?id=0B-_4IWxXwazQbWE5YmtqUWx3RVE) for design docs and
 meeting notes, and [this Zenhub board](https://app.zenhub.com/workspace/o/humancellatlas/data-store) to track our GitHub work.
 
 ## Overview
 
----
-
 The DSS is a replicated data storage system designed for hosting large sets of scientific experimental data on
 [Amazon S3](https://aws.amazon.com/s3/) and [Google Storage](https://cloud.google.com/storage/). The DSS exposes an API
 for interacting with the data and is built using [Chalice](https://github.com/aws/chalice),
 [API Gateway](https://aws.amazon.com/api-gateway/) and [AWS Lambda](https://aws.amazon.com/lambda/). The API also
 implements [Step Functions](https://aws.amazon.com/step-functions/) to orchestrate Lambdas for long-running tasks such
 as large file writes. You can find the API documentation and give it a try [here](https://dss.data.humancellatlas.org/).
 
-#### [Architectural Diagram](https://www.lucidchart.com/documents/view/b65c8898-46e3-4560-b3b2-9e85f1c0a4c7)
+### Architectural Diagram
 
-![DSS Sync SFN diagram](https://www.lucidchart.com/publicSegments/view/43dfe33a-47c9-466b-9cb6-6d941a406d8f/image.png)
+[![DSS Sync SFN diagram](https://www.lucidchart.com/publicSegments/view/43dfe33a-47c9-466b-9cb6-6d941a406d8f/image.png)](https://www.lucidchart.com/documents/view/b65c8898-46e3-4560-b3b2-9e85f1c0a4c7)
 
-#### DSS API
+### DSS API
 
 The DSS API uses [Swagger](http://swagger.io/) to define the [API specification](dss-api.yml) according to the
 [OpenAPI 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
 [Connexion](https://github.com/zalando/connexion) is used to map the API specification to its implementation in Python.
 
 You can use the
-[Swagger Editor](http://editor.swagger.io/#/?import=https://raw.githubusercontent.com/HumanCellAtlas/data-store/master/dss-api.yml)
+[Swagger Editor](http://editor.swagger.io/#/?import=https://raw.githubusercontent.com/DataBiosphere/data-store/master/dss-api.yml)
 to review and edit the API specification. When the API is live, the spec is also available at `/v1/swagger.json`.
 
```

Comment:
maybe keep a link to https://dss.dev.ucsc-cgp-redwood.org ?

Reply:
done in 
70d771d